src/app/components/pages/Witnesses.jsx

```diff
@@ -115,7 +123,7 @@ class Witnesses extends React.Component {
                             </a>
                         </span>
                     </td>
-                    <td>
+                    <td style = { witnessIsDisabled || noPriceFeed ? { textDecoration: 'line-through' } : null }>
```

Comment:
Needs to be in a stylesheet.

Reply:
@netuoso - The intent of this change is a good one. We have [new contribution guidelines](https://github.com/steemit/condenser/blob/master/CONTRIBUTING.md#enhancement-suggestions). Please check them out and follow them for future PRs.

We have a lot of work coming down the pipe for the steemit team. For the next couple months feature adds by community members will be slower to be integrated, and PR reviews will need to be corrected by the contributors somewhat quickly or risk being closed (likely to be revisited later)